### PR TITLE
feat: Reset DBusItem Properites before applying new changes

### PR DESCRIPTION
**Pull request by BlankParticle**
Fixes: #482 

More info in the issue.

### CODE REVIEW

File: dbusMenu.js [hunk 1]
```diff
@@ -130,6 +130,12 @@ export class DbusMenuItem extends Signals.EventEmitter {
         this.emit('property-changed', prop, this.propertyGetVariant(prop));
     }
 
+    propertiesReset() {
+        for (const [prop, value] of Object.entries(PropertyStore.DefaultValues)) {
+            this.propertySet(prop, value);
+        }
```
Comment (3v1n0) [inline dbusMenu.js:136]: nit:

```suggestion
        Object.entries(PropertyStore.DefaultValues).forEach(([prop, value]) =>
            this.propertySet(prop, value));
```

**Comment by BlankParticle** [comment]
Hello, @3v1n0, seems like you had been busy lately.

I would much appricate if you can look into and merge this PR into the extension. It is a much needed fix in this extension.

Thanks (and sorry for disturbing).

**Comment by 3v1n0** [comment]
Sorry, for delay, thanks!

**Comment by ztefn** [comment]
@3v1n0 Apparently too late for the LTS... looking forward to 10 year support for broken appindicators! /s

**Comment by 3v1n0** [comment]
You're very wrong, the fix is already in the 24.04 review queue since day 0

**Pull request merged**